//package core.io.protocol;

message DceQueryUserInfo
{
	optional string  uid = 1;
}

